pkg/api/v20230401/openshiftcluster_validatestatic.go

```diff
@@ -346,6 +365,24 @@ func (sv openShiftClusterStaticValidator) validateIngressProfile(path string, p
 	return nil
 }
 
+func (sv openShiftClusterStaticValidator) validateClusterResourceGroupTags(path string, t *Tags) error {
+	if len(*t) > maxTags {
+		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, path, fmt.Sprintf("The provided set of cluster resource group tags is too large; it can contain at most %d tags.", maxTags))
+	}
+
+	for k, v := range *t {
+		if !sv.clusterResourceGroupTagIsValid(k, v) {
+			return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, path, "One or more of the provided cluster resource group tags are invalid.")
```

Comment:
Are the rules for RG naming exposed in some msft docs we can provide along with the error message?

Reply:
This part is validating tags, not the RG name. But to answer the question with respect to tags, the rules are _not_ exposed in MSFT docs (see my response to your other question about the regex for validating tags).

I could definitely still add the rules into the error message though.

Reply:
I've updated this error message now.